ASIM/dev/ASimTester/ASimTester.csv

```diff
@@ -1120,4 +1120,55 @@ ActorScope,string,Optional,ProcessEvent,,,
 TargetOriginalUserType,string,Optional,ProcessEvent,,,
 TargetScopeId,string,Optional,ProcessEvent,,,
 TargetScope,string,Optional,ProcessEvent,,,
-
+User,string,Alias,UserManagement,,,ActorUserName
+Hostname,string,Alias,UserManagement,,,DvcHostname
+IpAddr,string,Alias,UserManagement,,,ScrIpAddr
+UpdatedPropertyName,string,Alias,UserManagement,,MultipleProperties|UpdatedPropertyName,EventSubType
+ActorUserType,string,Optional,UserManagement,Enumerated,Regular|Machine|Admin|System|Application|Service Principal|Other,
+ActorUsernameType,string,Conditional,UserManagement,Enumerated,UPN|Windows|DN|Simple,ActorUsername
+EventType,string,Mandatory,UserManagement,Enumerated,UserCreated|UserDeleted|UserModified|UserLocked|UserUnlocked|UserDisabled|UserEnabled|PasswordChanged|PasswordReset|GroupCreated|GroupDeleted|GroupModified|UserAddedToGroup|UserRemovedFromGroup|GroupEnumerated|UserRead|GroupRead,
+EventResult,string,Mandatory,UserManagement,Enumerated,Success|Failure,
+EventSeverity,string,Mandatory,UserManagement,Enumerated,Informational|Low|Medium|High,
+EventSchema,string,Mandatory,UserManagement,,UserManagement,
+EventSchemaVersion,string,Mandatory,UserManagement,SchemaVersion,,
+ActingAppId,string,Optional,UserManagement,,,
+ActiveAppName,string,Optional,UserManagement,,,
+ActingAppType,string,Optional,UserManagement,Enumerated,Process|Browser|Resource|Other,
+HttpUserAgent,string,Optional,UserManagement,Useragent,,
+ActorUsername,string,Mandatory,UserManagement,Username,,
+ActorUserIdType,string,Conditional,UserManagement,Enumerated,SID|UID|AADID|OktaId|AWSId,ActorUserId
+ActorUserId,string,Optional,UserManagement,,,
+ActorSessionId,string,Optional,UserManagement,,,
+EventSubType,string,Optional,UserManagement,Enumerated,UserRead|UserCreated|GroupCreated|UserModified|GroupModified,
+GroupId,string,Optional,UserManagement,,,
+GroupIdType,string,Conditional,UserManagement,Enumerated,SID|UID,GroupId
+GroupName,string,Optional,UserManagement,,,
+GroupNameType,string,Conditional,UserManagement,Enumerated,UPN|Windows|DN|Simple,GroupName
+GroupType,string,Optional,UserManagement,Enumerated,Local Distribution|Local Security Enabled|Global Distribution|Global Security Enabled|Universal Distribution|Universal Security Enabled|Other,
+GroupOriginalType,string,Optional,UserManagement,,,
+SrcFQDN,string,Optional,UserManagement,FQDN,,
+SrcDvcId,string,Optional,UserManagement,,,
+SrcDvcScopeId,string,Optional,UserManagement,,,
+SrcDvcScope,string,Optional,UserManagement,,,
+SrcDvcIdType,string,Optional,UserManagement,Enumerated,AzureResourceId|MDEid,
+SrcDeviceType,string,Optional,UserManagement,Enumerated,Computer|Mobile Device|IOT Device|Other,
+SrcGeostring,string,Optional,UserManagement,,,
+SrcGeostring,string,Optional,UserManagement,,,
```

Comment:
I think you meant SrcGeoCountry and SrcGeoRegion here.

Reply:
updated